codes/quantum/qudits/quantum_reed_solomon.yml

```diff
@@ -0,0 +1,30 @@
+#######################################################
```

Comment:
This is code_id: polynomial for regular quditsand code_id: galois_polynomial for galois qudits. Please delete this entry.